src/scripts/useredit.js

```diff
@@ -21,11 +21,23 @@ define(["jQuery", "loading", "libraryMenu", "fnchecked"], function($, loading, l
         })
     }
 
+    function loadPasswordResetProviders(page, user, providers) {
+        providers.length > 1 && !user.Policy.IsAdministrator ? page.querySelector(".fldSelectPasswordResetProvider").classList.remove("hide") : page.querySelector(".fldSelectPasswordResetProvider").classList.add("hide");
```

Comment:
wtf? :) I mean, don't manually write uglified code, we're trying to de-uglify the codebase ;)

```suggestion
        if (providers.length > 1 && !user.Policy.IsAdministrator) {
            page.querySelector(".fldSelectPasswordResetProvider").classList.remove("hide");
        } else {
            page.querySelector(".fldSelectPasswordResetProvider").classList.add("hide");
        }
```
Yes, it's not a one-liner, but it's _way_ easier to read.

Also, why check if user is admin?

Reply:
This is cloned from the select userauth provider which is also admin only it seemed appropriate to keep this option admin only for now as well